subgraph/abis/FleekNFA.json

```diff
@@ -0,0 +1,682 @@
+[
```

Comment:
I think this can be got also from deployments folder.

Reply:
It's autogenerated. The main abi file is in artifacts/contracts/FleekERC721.sol/FleekERC721.json but subgraph needs to store it separately.